### ISSUE TITLE
[MWAR-179] <packagingExcludes> should accept regexp to include/exclude specified jars

### ISSUE THREAD

**Issue by jira-importer** [apache/maven-war-plugin#354]
**[Matthew Grzechocinski](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=grzechom)** opened **[MWAR-179](https://issues.apache.org/jira/browse/MWAR-179?redirect=false)** and commented

It will be nice to be able to pass regular expression fo \<packagingExcludes>.
Now we can only specify WEB-INF/lib/*.jar and it will exclude all jars. What if I have o profile in which I want to exclude all external libraries which are stored on server (and there are many of them) but I want to include submodules of my application. Copying dependencies to profile with \<scope>provided\</scope> creates my pom really messy.

Summing up, It will be nice to set \<packagingExcludes>^WEB-INF/lib/prefix*.jar\<packagingExcludes> to exclude all jars except those which starts with prefix.
Can I expect this feature in the nearest future?

---

**Affects:** 2.1-alpha-2

**Issue Links:**
- [MWAR-81](https://issues.apache.org/jira/browse/MWAR-81) Request enhancement to pattern matching for packagingIncludes/packagingExcludes functionality (regular expressions?)
 (_**"duplicates"**_)
